inc/domainrecord.class.php

```diff
@@ -205,14 +227,37 @@ private function prepareInput($input) {
       }
 
       if (!isset($input['status']) || empty($input['status'])) {
-         $input['Status'] = self::STATUS_ACTIVE;
+         $input['status'] = self::STATUS_ACTIVE;
+      }
+
+      if (isset($input['domainrecordtypes_id'])) {
```

Comment:
This will not prevent update of an existing record from massive actions, as `domainrecordtypes_id` field will not be passed in input.